### PR TITLE
Update bindep.txt for rhel-8 platform

### PR DESCRIPTION
**Pull request by pabelanger**
For downstream builds, we are using python38 as the version of python.
This means, some packages like netaddr do not exist. These should be
installed via requirements files.

Signed-off-by: Paul Belanger <pabelanger@redhat.com>

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: bindep.txt [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
-python3-netaddr [platform:rpm]
-python3-jmespath [platform:rpm]
-python3-passlib [platform:rpm epel]
-qemu-img [platform:rpm]
+gcc [compile platform:centos-8 platform:rhel-8]
+libcurl-devel [compile platform:centos-8 platform:rhel-8]
+libxml2-devel [compile platform:centos-8 platform:rhel-8]
+openssl-devel [compile platform:centos-8 platform:rhel-8]
+python38-devel [compile platform:centos-8 platform:rhel-8]
+qemu-img [platform:centos-8 platform:rhel-8]
```
Comment (pabelanger) [inline bindep.txt:6]: Is this needed on the controller side? It seems like it might be for the remote node

Comment (mnecas) [reply bindep.txt:6]: We need it in the ovirt_disk module. https://github.com/oVirt/ovirt-ansible-collection/blob/master/plugins/modules/ovirt_disk.py#L599

**Comment by pabelanger** [comment]
@mnecas do you think we can get this merged and tag new release for galaxy / AH?

**Review by mnecas — APPROVED** [review]
LGTM

**Comment by pabelanger** [comment]
@mnecas thanks! Do you happen to know an ETA on publishing to galaxy / AH?

**Comment by mnecas** [comment]
@pabelanger we don't have any specific timeline for the galaxy so can try next week,
as for AH, we release it with RHV so it will take a bit longer

**Comment by pabelanger** [comment]
okay cool, the rhv release would be helpful for downstream execution environments. I'm sure we'll be in touch next week :)

**Pull request merged**